bitnami/redis/Chart.yaml

```diff
@@ -24,4 +24,4 @@ maintainers:
 name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
-version: 16.5.4
+version: 16.5.5
```

Comment:
Could you bump the minor version? It is adding a non-breaking feature

Reply:
Just for future reference, when do you guys prefer the patch version change?